wf-galleon/WFLY-11989_Galleon_Maven_Repository.adoc

```diff
@@ -0,0 +1,90 @@
+= WFLY-11989: wildfly-dist should generate and deploy a zipped maven repo for galleon
+:author:            Jean-Francois Denise
+:email:             jdenise@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+Offer the ability to package all Galleon required artifacts to provision WildFly server.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/WFLY-11989[WFLY-11989]
+
+=== Related Issues
+
+* https://issues.jboss.org/browse/EAP7-891[EAP7-891]
+
+* https://issues.jboss.org/browse/EAP7-1218[EAP7-1218]
+
+* https://issues.jboss.org/browse/WFGP-132[WFGP-132] (Obsolete, not followed approach)
+
+* https://issues.jboss.org/browse/WFGP-135[WFGP-135]
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+=== Testing By
+// Put an x in the relevant field to indicate if testing will be done by Engineering or QE. 
+// Discuss with QE during the Kickoff state to decide this
+[ ] Engineering
+
+[ ] QE
+
+=== Affected Projects or Components
+
+* OpenShift
+
+=== Other Interested Projects
+
+* WildFly Galleon
+
+* Offliner: https://github.com/release-engineering/offliner
+
+== Requirements
```

Comment:
@jfdenise What artefacts do we expect to be present in Maven repository provisioned in this plugin compared to the zipped Maven product repository?

Reply:
@mjurc we will have *.pom[.sha|.md5] and *.jar[.sha|.md5] No jar file containing sources. So dependencies needed to build application will be not present.

Reply:
It would contain artifacts required for server runtime (=feature packs and their dependencies). Zipped product repository contains also artifacts required for development / BOMs / quickstarts